src/test/java/teammates/test/cases/testdriver/BackDoorTest.java

```diff
@@ -473,7 +484,17 @@ private void verifyPresentInDatastore(AccountAttributes expectedAccount) {
         expectedAccount.studentProfile.modifiedDate = actualAccount.studentProfile.modifiedDate;
         assertEquals(gson.toJson(expectedAccount), gson.toJson(actualAccount));
     }
-    
+
+    private void verifyPresentInDatastore(FeedbackQuestionAttributes expectedQuestion) {
+        String questionJsonString = BackDoor.getFeedbackQuestionAsJson(expectedQuestion.feedbackSessionName,
+                                        expectedQuestion.courseId, expectedQuestion.questionNumber);
```

Comment:
Just to nitpick, you can try aligning the parameters. 

See
https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true#h.ukkp0ov105ly
on 
<code>Use line wrapping to improve readability.</code>